src/components/VisitsPanelList/index.js

```diff
@@ -69,6 +69,18 @@ const VisitsPanelList = ({ visits, title, showButtons }) => {
                             Start
                           </button>
 
+                          <button
+                            className="nhsuk-button nhsuk-u-margin-right-5 nhsuk-button--secondary"
+                            onClick={() => {
+                              Router.push({
+                                pathname: "/wards/visits/[id]/edit",
+                                query: { id: visit.id },
```

Comment:
Interesting fact: `pathname` is not what I thought it was, it's the path inside our `pages` directory. Originally I did `/wards/visits/${visit.id}/edit` but that threw 404 errors in the console.

https://github.com/vercel/next.js/issues/2208#issuecomment-341939582